contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -140,6 +140,8 @@ var ContentNodeModel = BaseModel.extend({
 		children:[],
 		tags:[],
 		assessment_items:[],
+		metadata: {"resource_size" : 0, "resource_count" : 0},
```

Comment:
Defaults so that channel page can function without saving to the db automatically